File: workspace/data-model/src/main/java/de/pfiva/data/model/Topic.java
package de.pfiva.data.model;

import java.io.Serializable;

public class Topic implements Serializable {

	private static final long serialVersionUID = 1L;
	
	private int id;
	private String topicname;
	private String creationDate;
	private String modificationDate;
	
	private int surveyCount;
	private int messageCount;
	
	public int getId() {
		return id;
	}
	public void setId(int id) {
		this.id = id;
	}
	public String getTopicname() {
		return topicname;
	}
	public void setTopicname(String topicname) {
		this.topicname = topicname;
	}
	public String getCreationDate() {
		return creationDate;
	}
	public void setCreationDate(String creationDate) {
		this.creationDate = creationDate;
	}
	public String getModificationDate() {
		return modificationDate;
	}
	public void setModificationDate(String modificationDate) {
		this.modificationDate = modificationDate;
	}
	public int getSurveyCount() {
		return surveyCount;
	}
	public void setSurveyCount(int surveyCount) {
		this.surveyCount = surveyCount;
	}
	public int getMessageCount() {
		return messageCount;
	}
	public void setMessageCount(int messageCount) {
		this.messageCount = messageCount;
	}
	@Override
	public String toString() {
		return "Topic [id=" + id + ", topicname=" + topicname + ", creationDate=" + creationDate + ", modificationDate="
				+ modificationDate + ", surveyCount=" + surveyCount + ", messageCount=" + messageCount + "]";
	}
}

File: angular_workspace/pfiva-web/src/app/data-model/Topic.ts
export class Topic {

	private id: number;
	private topicname: string;
	private creationDate: string;
	private modificationDate: string;
	private surveyCount: number;
	private messageCount: number;

	public get Id(): number {
		return this.id;
	}

	public set Id(value: number) {
		this.id = value;
	}
	public get Topicname(): string {
		return this.topicname;
	}

	public set Topicname(value: string) {
		this.topicname = value;
	}
	public get CreationDate(): string {
		return this.creationDate;
	}

	public set CreationDate(value: string) {
		this.creationDate = value;
	}
	public get ModificationDate(): string {
		return this.modificationDate;
	}

	public set ModificationDate(value: string) {
		this.modificationDate = value;
	}
	public get SurveyCount(): number {
		return this.surveyCount;
	}

	public set SurveyCount(value: number) {
		this.surveyCount = value;
	}
	public get MessageCount(): number {
		return this.messageCount;
	}

	public set MessageCount(value: number) {
		this.messageCount = value;
	}
}